.github/workflows/test-sui.yaml

```diff
@@ -36,18 +36,20 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup Node.js
-        uses: actions/setup-node@v4
+        uses: axelarnetwork/axelar-cgp-sui/.github/actions/nodejs-ci@main
         with:
-          node-version: '18'
-          cache: 'npm'
+          nodejs-version: '18' # Hardcoded to ensure consistency.
 
-      - name: Install dependencies
-        run: npm ci
+      - name: Get SUI Version
+        shell: bash
+        run: |
+          SUI_VERSION=$(jq -r '.SUI_VERSION' "node_modules/@axelar-network/axelar-cgp-sui/version.json")
+          echo "SUI_VERSION=$SUI_VERSION" >> $GITHUB_ENV
 
       - name: Setup Sui CLI
-        uses: axelarnetwork/axelar-cgp-sui/.github/actions/install-sui-cli@main
+        uses: axelarnetwork/axelar-cgp-sui/.github/actions/setup-sui@main
         with:
-          sui-version: $(jq -r '.SUI_VERSION' node_modules/@axelar-network/axelar-cgp-sui/version.json)
+          sui-version: ${{ env.SUI_VERSION }}
```

Comment:
what's the reason to create a separate step instead of calling jq directly? style? not a strong opinion if you prefer to separate it